test/fixtures/flow/type-annotations/136/actual.js

```diff
@@ -0,0 +1,4 @@
+type A = {
+	p: {},
+	...{},
+};
```

Comment:
Yeah, PR for astexplorer hasn't been merged yet https://github.com/fkling/astexplorer/pull/208